docs/DeveloperGuide.adoc

```diff
@@ -447,6 +447,73 @@ In other words, when RestaurantBook#deleteMember() or RestaurantBook#deleteIngre
 
 * Decision: Alternative 1 was adopted as it is an easier implementation and it makes it more convenient for the user to delete a record. It is intuitive that if a deleteMember or deleteIngredient command is called and these objects cease to exist in the restaurant book, the corresponding Booking or Recipe containing these objects should also cease to exist. To deal with the possibility that the member or ingredient is accidentally deleted, and all referenced bookings and recipes is unintentionally deleted, an additional field in the delete command can be added as a confirmation that these records should be deleted.
 
+=== The `stats` command
+
+When the `stats` command is entered, the current statictis of the amount of bookings is to be displayed to the user in the form of a bar chart.
```

Comment:
"statistics"- fix typo
"amount of bookings"- what does this even mean

@limli bear in mind you'll have to explain this in the user guide too

if you're unsure what you want right now, probably ok. but remember to give a clear specification of what the command does.